### PR TITLE
Agrege 2 cursos en la seccion free-course-es.md uno en sql y otro en git

### PR DESCRIPTION
**Pull request by MahonriM**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-es.md [hunk 1]
```diff
@@ -86,7 +86,7 @@
 ### Control de Versiones
 
 * [Git & GitHub](https://www.pildorasinformaticas.es/course/curso-de-git-github) - Juan Díaz (Píldoras Informáticas)
-
+* [Git & GitHub](https://www.youtube.com/watch?v=L_lWQZNhN7w&list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls) -  Ignacio Gutiérrez  (BlueWeb)
```
Comment (davorpa) [inline courses/free-courses-es.md:89]:
```suggestion
* [Curso: GIT - GITHUB \[Desde Cero\]](https://www.youtube.com/playlist?list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls) - Ignacio Gutiérrez (BlueWeb)
* [Git & GitHub](https://www.pildorasinformaticas.es/course/curso-de-git-github) - Juan Díaz (Píldoras Informáticas)
```

File: courses/free-courses-es.md [hunk 2]
```diff
@@ -86,6 +86,7 @@
 ### Control de Versiones
 
 * [Git & GitHub](https://www.pildorasinformaticas.es/course/curso-de-git-github) - Juan Díaz (Píldoras Informáticas)
+* [Git & GitHub](https://www.youtube.com/playlist?list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls)  - Ignacio Gutiérrez (BlueWeb)
```
Comment (davorpa) [inline courses/free-courses-es.md:89]:
```suggestion
* [Git & GitHub](https://www.youtube.com/playlist?list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls) - Ignacio Gutiérrez (BlueWeb)
```

File: courses/free-courses-es.md [hunk 3]
```diff
@@ -49,6 +49,7 @@
 * [Fundamentos de las bases de datos (2011)](https://ocw.ua.es/es/ingenieria-y-arquitectura/fundamentos-de-las-bases-de-datos-2011.html)
 * [Manual práctico de SQL](https://www.lawebdelprogramador.com/cursos/archivos/ManualPracticoSQL.pdf) - Álvaro E. García (PDF)
 * [Principios de SQL](https://programadorwebvalencia.com/cursos/sql/introducci%C3%B3n/) - Andros Fenollosa (Programador Web Valencia)
+* [Crear funciones en SQL Server](https://mahonrim.github.io/FuctionswithSQLServer/) - Mahonri Martinez (Programador Web/Estudiante de IT)
```
Comment (davorpa) [inline courses/free-courses-es.md:52]: Este contenido no es realmente valioso

Delete patch
```suggestion
```

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-es.md [hunk 4]
```diff
@@ -86,6 +86,7 @@
 ### Control de Versiones
 
 * [Git & GitHub](https://www.pildorasinformaticas.es/course/curso-de-git-github) - Juan Díaz (Píldoras Informáticas)
+* [Git & GitHub](https://www.youtube.com/playlist?list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls) - Ignacio Gutiérrez (BlueWeb)
```
Comment (LuigiImVector) [inline courses/free-courses-es.md:89]:
```suggestion
* [Git & GitHub](https://www.youtube.com/playlist?list=PLPl81lqbj-4I8i-x2b5_MG58tZfgKmJls) - Ignacio Gutiérrez, BlueWeb (YouTube)
```

"BlueWeb" isn't the hosted platform but the author

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**